course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -215,91 +401,165 @@
     "\n",
-    " * the air temperature in the E1 scenario in 2099, \n",
-    " * the air temperature in the A1B scenario in 2099, and\n",
+    "Plot the following data in a single figure with three maps, side-by-side in one row :\n",
+    "\n",
+    " * the air temperature in the E1 scenario for the year 2099, \n",
+    " * the air temperature in the A1B scenario for the year 2099, and\n",
     " * the difference between the two scenarios.\n",
     "\n",
-    "Think about the most appropriate matplotlib colormap(s) to use for each plot. Hint: the different matplotlib colormaps can be seen at https://matplotlib.org/1.5.3/examples/color/colormaps_reference.html. "
+    "Think about the most appropriate matplotlib colormap(s) to use for each plot.\n",
+    "\n",
+    "Hint: the different matplotlib colormaps can be seen at https://matplotlib.org/1.5.3/examples/color/colormaps_reference.html. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "2b\\. What information do your plots show? "
+    "# user code ..."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.3b"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "3\\. Produce cubes that describe the area-averaged air temperature over time for each scenario. Calculate the model difference between these two cubes."
+    "----\n",
+    "\n",
+    "#### 3\\. Plot E1, A1B and difference side by side\n",
+    "\n",
+    "Plot the data in a single figure with three maps, side-by-side in one row :\n",
+    "\n",
+    " * the air temperature in the E1 scenario for the year 2099, \n",
+    " * the air temperature in the A1B scenario for the year 2099, and\n",
+    " * the difference between the two scenarios.\n",
+    "\n",
+    "Think about the most appropriate plot functions, and the matplotlib colormap(s) to use for each plot.\n",
+    "\n",
+    "Hint: the different matplotlib colormaps can be seen at https://matplotlib.org/1.5.3/examples/color/colormaps_reference.html. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# user code ..."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.3c"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "4\\. Produce a single line plot with the data from the two cubes you produced in part 3. Make sure you label the lines you plot."
+    "----\n",
+    "\n",
+    "#### 3. Produce time sequences of regional average air temperatures\n",
```

Comment:
This numbering needs to be updated, this should be question 4.

I also much prefer this question.